### PR TITLE
feature/CSI-1764_resolve_1695

### PR DESCRIPTION
**Pull request by ArbelNathan**
resolving issues in https://github.com/IBM/ibm-block-csi-driver/pull/218

### CODE REVIEW

File: node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go [hunk 1]
```diff
@@ -501,7 +501,8 @@ func NewGetDmsPathHelperGeneric(executer executer.ExecuterInterface) GetDmsPathH
 
 func (o GetDmsPathHelperGeneric) WaitForDmToExist(volumeUuid string, maxRetries int, intervalSeconds int) (string, error) {
 
-	args := []string{"show", "maps", "raw", "format", "\"", "%d" + MpathdSeparator + "%w", "\""}
+	formatOrder := strings.Join([]string{"%d", "%w"}, MpathdSeparator)
```
Comment (oriyarde) [inline node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go:504]: IMO it's not just about order, but also which fields to show.
maybe just `format`?

Comment (ArbelNathan) [reply node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go:504]: maybe `formatTemplate`?

Comment (oriyarde) [reply node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go:504]: sure

Comment (ArbelNathan) [reply node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go]: done

**Review by oriyarde — APPROVED** [review]
LGTM

**Review by matancarmeli7 — APPROVED** [review]

**Comment by matancarmeli7** [comment]
LGTM

**Pull request merged**